README.md

```diff
@@ -285,6 +282,112 @@ func main() {
 }
 ```
 
+For use SQL to query to tarantool, call `Execute`. 
+
+Pay an attention, that with the different types of query (DDL, DQL, DML etc.),
```

Comment:
"Pay attention that with different types of queries (DDL, DQL, DML etc.) some fields of the response structure (`MetaData` and `SQLInfo`) may be empty."

Reply:
Fixed.